openedx/openedx-platform
# (BB-1389) Update enrollment serializer and Add problem submission history endpoint

Introduction
------------

Another description as I don't have permission to edit the above.

This PR adds an endpoint displaying the submission history for each course a specific user is enrolled.

**Sandbox URL**: https://pr20948.sandbox.opencraft.hosting/ (provisioning)

Testing instructions
--------------------

These can use the sandbox above or a devstack.

1. Ensure the logged in user has submitted a few problems.
2. Browse http://localhost:18000/api/enrollment/v1/submission_history or http://pr20948.sandbox.opencraft.hosting/api/enrollment/v1/submission_history
3. Make sure a JSON object showing the user submissions is displayed

Screenshots
-------------
![image](https://user-images.githubusercontent.com/5691347/77781993-4b78b180-7035-11ea-9cfe-3e3840f25c14.png)



Author notes and concerns
-------------

1. The update in the enrollment serializer makes extra mongodb calls.
